From: Markus Koschany <apo@debian.org>
Date: Fri, 30 Mar 2018 18:15:54 +0200
Subject: java9

Don't use wildcards to avoid an ambiguous reference to java.lang.Module.

Bug-Debian: https://bugs.debian.org/893239
Forwarded: no
---
 src/mapper/java/org/codehaus/jackson/map/module/SimpleModule.java | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/mapper/java/org/codehaus/jackson/map/module/SimpleModule.java b/src/mapper/java/org/codehaus/jackson/map/module/SimpleModule.java
index c36d0a8..39936c2 100644
--- a/src/mapper/java/org/codehaus/jackson/map/module/SimpleModule.java
+++ b/src/mapper/java/org/codehaus/jackson/map/module/SimpleModule.java
@@ -4,7 +4,10 @@ import java.util.HashMap;
 import java.util.Map;
 
 import org.codehaus.jackson.Version;
-import org.codehaus.jackson.map.*;
+import org.codehaus.jackson.map.Module;
+import org.codehaus.jackson.map.JsonDeserializer;
+import org.codehaus.jackson.map.JsonSerializer;
+import org.codehaus.jackson.map.KeyDeserializer;
 import org.codehaus.jackson.map.deser.ValueInstantiator;
 
 /**
